src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -1348,5 +1347,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Protected Overrides Function CommonWithCheckOverflow(checkOverflow As Boolean) As CompilationOptions
             Return WithOverflowChecks(checkOverflow)
         End Function
+
+        Public Overrides Property NullableContextOptions As NullableContextOptions
+            Get
+                Return NullableContextOptions.Disable
+            End Get
+            Protected Set(value As NullableContextOptions)
+                Throw ExceptionUtilities.Unreachable
```

Comment:
>Unreachable [](start = 41, length = 11)

NotImplemented seems more suitable, since this is a public API (someone can reach it)